pallets/xcm-helper/src/lib.rs

```diff
@@ -819,4 +737,41 @@ pub mod pallet {
 			whitelisted_tokens.contains(&asset_id)
 		}
 	}
+
+	impl<T: Config> TheaIncomingExecutor for Pallet<T> {
+		fn execute_deposits(_network: Network, deposits: Vec<u8>) {
+			let deposits = BoundedVec::<ApprovedWithdraw, ConstU32<10>>::decode(&mut &deposits[..])
+				.unwrap_or_default();
+
+			for deposit in deposits {
+				let deposit_request = match ParachainDeposit::decode(&mut &deposit.payload[..]) {
+					Ok(deposit_) => deposit_,
+					Err(_) => continue,
```

Comment:
Why is it just silenced?

Reply:
that function doesn’t return the result. So there is no way we can handle it. Also, it should not fail.